### ISSUE TITLE
Unable to call `array.push(123)`, where`array` is an `Array<number>`

### ISSUE THREAD

**Issue by trusktr** [JSMonk/hegel#290]
Is this a bug, or did I miss a feature of the type safety?

https://hegel.js.org/try#DYUwLgBAdgXBCCAnRBDAngHigVwLYCMREA+CAXggG0AGAGgmoF0AoKAOgAdsBnACwAoAjACYAzAEogA

**Comment by JSMonk** [comment]
Yes and no. We catch an effect which was not handled, but it's useless effect :D 
We are thinking about, how to solve this problem with `Array#push`. Because it is so common to use `push` without the returned result of the function. Thank you for the issue :3

**Comment by JSMonk** [comment]
A brief offtopic,  I want to make Hegel available for https://github.com/trusktr/solid project (it will be the first Framework which I will support with high-level static type checking), but I will need some help after porting DOM API. Can I ask you to help me with this task? If it will be interesting for you, where can I message you to discuss this idea?

**Comment by trusktr** [comment]
That'd be great! The author of Solid is @ryansolid, and I built a custom element system on top of that at http://github.com/lume/element. The Solid chat server is at https://discord.gg/Up5bYGF, where you can find us.
